llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1209,18 +1219,13 @@ void MCCASBuilder::startSection(const MCSection *Sec) {
 
   CurrentSection = Sec;
   CurrentContext = &SectionContext;
-  const MCSection *DebugInfoSection =
-      Asm.getContext().getObjectFileInfo()->getDwarfInfoSection();
-
-  auto DwarfLineSection =
-      Asm.getContext().getObjectFileInfo()->getDwarfLineSection();
 
   if (RelocLocation == Atom) {
     // Build a map for lookup.
     for (auto R : ObjectWriter.getRelocations()[Sec]) {
       // For the Dwarf Sections, just append the relocations to the
       // SectionRelocs. No Atoms are considered for this section.
-      if (R.F && DwarfLineSection != Sec && Sec != DebugInfoSection)
+      if (R.F && DwarfSections.Line != Sec && Sec != DwarfSections.DebugInfo)
```

Comment:
Small nitpick, but can you change this to either be 
`if (R.F && Sec != DwarfSections.Line && Sec != DwarfSections.DebugInfo)` or `if (R.F && DwarfSections.Line != Sec && DwarfSections.DebugInfo != Sec)` so it looks consistent?